[docs] Remove NoSsr where possible (#15510)
mui-org/material-ui: docs/src/pages/demos/badges/badges.md

{{"demo": "pages/demos/badges/BadgeVisibility.js"}}

> ## Customized badge

Author: @mbrookes The capitalization of the documentation headers is inconsistent. Do you agree with this format?

Reviewer: I guess it depends whether badge refers to the UI element (a badge) or the React component `Badge`. :)

Author: I think that it's better to refer to the UI element. It might be composed a multiple React components.

Reviewer: In which case, 
```suggestion
## Customized badges
```
reads better.

Author: 👍

{{"demo": "pages/demos/badges/BadgeVisibility.js"}}

## Customized badges

If you have read the [overrides documentation page](/customization/overrides/)

> but you are not confident jumping in, here are some examples of how you can customize the component.

Reviewer 2: ```suggestion
and you are still not confident jumping in, here are some examples of how you can customize the component.
```
?

Author: 👌

Reviewer: Perhaps a hybrid?

```suggestion
but you are still not confident jumping in, here are some examples of how you can customize the component.
```

OTOH, I've never been to keen on the "not confident" – it's a bit condescending and "jumping in" feels like a misplaced idiom.

How about something like:

"Here are some examples of customizing the component. You can learn more about this in the [overrides documentation page](/customization/overrides/)."

A forward reference seems more natural than an assumptive back ref.

The "So you've been wondering X?" type phrases dotted around the docs feel a bit stilted in English. (No offence @oliviertassinari. Perhaps that's idiomatic in French?)

Reviewer 2: Hmm, I did have the same worry. I like your change :+1:
